frontend_lib/test/component/FileDropzone.spec.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import { expect } from 'chai'
+import { mount } from 'enzyme'
+import FileDropzone from '../../src/component/FileDropzone/FileDropzone.jsx'
+import { FILE_PREVIEW_STATE } from '../../src/helper.js'
+import sinon from 'sinon'
+require('../../src/component/FileDropzone/FileDropzone.styl')
+
+describe('<FileDropZone />', () => {
+  const onDropCallBack = sinon.spy()
+  const onClickCallBack = sinon.spy()
+
+  const props = {
+    onDrop: onDropCallBack,
```

Comment:
if you don't do any tests on these spies, you don't need them.
Replace them by an empty function